erpnext/hr/doctype/employee/employee.py

```diff
@@ -173,6 +173,26 @@ def validate_date(self):
 		elif self.contract_end_date and self.date_of_joining and (getdate(self.contract_end_date) <= getdate(self.date_of_joining)):
 			throw(_("Contract End Date must be greater than Date of Joining"))
 
+	def validated_suspension(self):
+		if self.status != "Suspended":
```

Comment:
You can reduce the number of times this runs by checking `doc_before_save`
```suggestion
		doc_before_save = self.get_doc_before_save()
		if self.status != "Suspended" and doc_before_save.status == "Suspended":
```

Reply:
done